cspell.config.js

```diff
@@ -11,9 +11,15 @@ module.exports = {
     words: [
         ...baseConfig.words,
         'dfinity',
+        'Distrikt',
```

Comment:
you only need the lowercase variant of each of these words, no need to duplicate them with different capitalizations